pages/fundamentals/data-durability.mdx

```diff
@@ -114,6 +114,10 @@ Alternatively, you can make one directly by running the following query:
 CREATE SNAPSHOT;
 ```
 
+<Callout type="info">
+If another snapshot is already being created or no committed writes to the database have been made since the last snapshot this query will fail with an error.
```

Comment:
```suggestion
If another snapshot is already being created or no committed writes to the database have been made since the last snapshot, this query will fail with an error.
```